content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,126 @@
+- :guilabel:`Official` is a production mode that sends your invoices directly to the Agenzia
+  dell'Entrate.
 
-Enable FatturaPA on the Sales journal
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Open your sales journal. Under the *Advanced Settings* tab, in the **Electronic invoicing** field,
-check *FatturaPA* and *Save*.
+.. warning::
+   The choice of the mode is **not reversible**. Once in :guilabel:`Official` mode it is not
+   possible to get back to the :guilabel:`Test` and :guilabel:`Demo`. Equally is not possible to go
+   from :guilabel:`Test` to :guilabel:`Official`.
 
-.. image:: italy/journal.png
-   :align: center
-   :alt: FatturaPA feature in the sales journal's settings
+After having selected the right mode, you need to accept the terms and conditions right below the
+mode selection. Once you have :guilabel:`Saved` the mode you want, you can start recording your
+Accounting transactions in Odoo.
 
 Issue invoices
---------------
+==============
 
-Send an invoice the way you would do it normally. The EDI process proceeds automatically.
+The **FatturaPA** feature is automatically actived, you can check it in
+:menuselection:`Configuration --> Journals --> Customer Invoices -->  Advanced Settings`
+You can create a new invoice by going to :menuselection:`Accounting dashboard --> New invoice`, once
+confirmed the e-invoice will be generated and sent.
 
-.. image:: italy/sending.png
-   :align: center
-   :alt: Asynchronous sending on a customer invoice
+.. warning::
+   The e-invoice is only sent to the government if you are in :guilabel:`Official` mode.
 
-You can check the current status of your customer invoice under the **Electronic invoicing** field.
+You can check the current status of your customer invoice under the :guilabel:`Electronic invoicing`
+field. The xml file is accessible in the chatter of the invoice.
 
 .. image:: italy/processing.png
    :align: center
    :alt: Electronic invoicing status (waiting for confirmation)
 
+Vendor bills
+============
+
+As for invoices, when creating a vendor bill, the :guilabel:`Fattura PA` option is automatically
+activated in the Vendor bill Journals' advanced settings.
+When inserting taxes in a vendor bill, you have the possibility to choose **reverse charge** taxes,
+these are automatically activated in the Italian fiscal position. By going to
+:menuselection:`Configuration --> Taxes`, you will see that the 10% & 22% services and goods taxes
+are activated and have the correct tax grids; these are set up automatically to ensure the correct
+booking of accounting entries and display of the tax report.
+Once the bill is confirmed, a banner appears, and the bill can be sent to the EDI service.
+
+Integration documents
+=====================
+
+If one of the two parties of the reverse charge is not Italian, it is required to send additional
+integration documents to the SDI. Depending on the case, these documents, **Dati Documento** can be
+type TD 16, TD 17, TD 18 and others.
```

Comment:
@lordkrandel Put this over here, but I'm not sure if we should integrate it with other sections, also any specifications about Odoo I should mention here?
Lastly, do you have a nice link for TD.. that I can put here? (if it can be useful)

Reply:
For the *Tax* integrations, I've written this spec about External Reverse Charge, that you can use.
You should be able to see the content - if not, just tell me and I will export it in a Google doc or sth
https://www.odoo.com/web#id=2823646&model=project.task

Reply:
I will integrate this in the next few hours with how to configure the invoice in Odoo to handle those cases, either here or (better) on the task itself. Will let you know when there's more.